### ISSUE TITLE
Create the Classifier

### ISSUE THREAD

**Issue by wgranger** [zooniverse/liberating-the-liberator#8]
At its core, this classifier will work similarly to the one on PFE without all the experimental bells and whistles.

**Comment by shaunanoordin** [comment]
The Classifier is pretty straightforward, but we can break this down into smaller independent tasks.

### Task 1: Fetch Subjects from Panoptes
* Goal: using the API client, get a random subject (as JSON) from Panoptes 
  * e.g. `panoptesApiClient.type('subjects').get('671189').then(...);`
* **Caveat:** For testing purposes, we'll need to _either..._
  * set up a new LtL project on `staging` (more setup work) or 
  * use `production`'s more "true-to-life" AnnoTate data (we need to switch our dev environment to production).

### Task 2: Display Subject
* Goals:
  * Given some subject data (ID, image URL, metadata), show the Subject (e.g. a journal page) visually on the web page.  
  * Allow the user to zoom/rotate/move the Subject.
* This will mostly involve SVG magic hooked up with keyboard/mouse/touch events.
  * I'll be using this for reference: https://github.com/shaunanoordin/zooniverse-react-transcriber
* For bonus points: keep ACCESSIBILITY in mind. Keyboard shortcuts FTW!

### Task 3: Send Classifications to Panoptes
* Goal: given some annotations, submit the Classification back to Panoptes
* In the future, we may also need to add extra metadata to the Classifications, e.g. the number of times a user accessed a collaborative feature. Don't worry about this for now, but keep it in mind.
* How would the annotations/Classifications be structured? I assume it'll be spelled out in the workflow, but this is worth checking.
* **Caveat:** we'll need to eventually check if the data we're feeding back is legit (valid, usable, and aggregatable) but this may be a bigger issue than just the Classifier.

### Task 4: Allow users to make Annotations
* Goal: allow users to mark start coordinates and end coordinates on a Subject, then enter some text.
* A corollary to Task 2 and Task 3.

### Task 0: Include Panoptes Javascript Client 
* Basically: https://github.com/zooniverse/panoptes-javascript-client
* We add `panoptes-client` `2.7.2` or [whatever](https://www.npmjs.com/package/panoptes-client) to `package.json`
* This task will take approximately 30 man hours and about 4 weeks of regression testing.

### Other
* Features such as a collaborative dropdown should be separated into their own issues. Let's keep this fairly simple.

**Comment by srallen** [comment]
Just an FYI the the [subject queue endpoint](http://docs.panoptes.apiary.io/#reference/subjects/subject-collection) is the one that should be used for fetching subjects for the subject queue for classification. Roger and I are discussing building a service worker for this for the v2 of the javascript client: https://github.com/zooniverse/rfc/issues/16#issuecomment-306459016

However, I'm not sure how or if that'll meet up with the timeline for this project. I'd just say keep it on your radar. 

Also for annotation convention for classifications, the API doesn't validate on a particular format, however, for axillary services (i.e. aggregation) we need to conform to a convention. It's documented on PFE's readme: https://github.com/zooniverse/Panoptes-Front-End#format-of-annotation-values
(Chris Synder and I made this mistake for the initial Wildcam build...)

But, aggregation is being rewritten this summer and there are new services like caesar, so it'd be good to check in with back end devs about annotation conventions.

**Comment by wgranger** [comment]
I feel like at its core, this has pretty much been implemented.
